x-pack/plugins/security_solution/public/flyout/right/components/threat_intelligence_overview.test.tsx

```diff
@@ -0,0 +1,350 @@
+  });
+
+  it('should render 0 field enriched', () => {
+    (useInvestigationTimeEnrichment as jest.Mock).mockReturnValue({
+      result: {
+        enrichments: [
+          {
+            'threat.indicator.file.hash.sha256': 'sha256',
+            'matched.atomic': ['sha256'],
+            'matched.field': ['file.hash.sha256'],
+            'matched.id': ['matched.id.1'],
+            'matched.type': ['indicator_match_rule'],
+          },
+        ],
+        totalCount: 1,
+      },
+      loading: false,
+    });
+
+    const { getByTestId } = render(
+      <TestProviders>
+        <RightPanelContext.Provider value={panelContextValue}>
+          <ThreatIntelligenceOverview />
+        </RightPanelContext.Provider>
+      </TestProviders>
+    );
+    expect(getByTestId(INSIGHTS_THREAT_INTELLIGENCE_CONTENT_TEST_ID)).toHaveTextContent(
+      '0 field enriched with threat intelligence'
+    );
+  });
+
+  it('should render 0 match detected', () => {
+    (useInvestigationTimeEnrichment as jest.Mock).mockReturnValue({
+      result: {
+        enrichments: [
+          {
+            'threat.indicator.file.hash.sha256': 'sha256',
+            'matched.atomic': ['sha256'],
+            'matched.field': ['file.hash.sha256'],
+            'matched.id': ['matched.id.2'],
+            'matched.type': ['investigation_time'],
+            'event.type': ['indicator'],
+          },
+        ],
+        totalCount: 1,
+      },
+      loading: false,
+    });
+
+    const { getByTestId } = render(
+      <TestProviders>
+        <RightPanelContext.Provider value={panelContextValue}>
+          <ThreatIntelligenceOverview />
+        </RightPanelContext.Provider>
+      </TestProviders>
+    );
+    expect(getByTestId(INSIGHTS_THREAT_INTELLIGENCE_CONTENT_TEST_ID)).toHaveTextContent(
+      '0 threat match detected'
+    );
+  });
+
+  it('should render loading', () => {
+    (useInvestigationTimeEnrichment as jest.Mock).mockReturnValue({
+      result: undefined,
+      loading: true,
+    });
+
+    const { getByTestId } = render(
+      <TestProviders>
+        <RightPanelContext.Provider value={panelContextValue}>
+          <ThreatIntelligenceOverview />
+        </RightPanelContext.Provider>
+      </TestProviders>
+    );
+    expect(getByTestId(INSIGHTS_THREAT_INTELLIGENCE_LOADING_TEST_ID)).toBeInTheDocument();
+  });
+
+  it('should only render null when eventId is null', () => {
+    (useInvestigationTimeEnrichment as jest.Mock).mockReturnValue({
+      result: {
+        enrichments: [],
+      },
+      loading: false,
+    });
+    const contextValue = {
+      ...panelContextValue,
+      eventId: null,
+    } as unknown as RightPanelContext;
+
+    const { container } = render(
+      <TestProviders>
+        <RightPanelContext.Provider value={contextValue}>
+          <ThreatIntelligenceOverview />
+        </RightPanelContext.Provider>
+      </TestProviders>
+    );
```

Comment:
hey, one nit / tip: you can have something like 
```
const renderTIOverview = (contextValue) => <TestProviders>
        <RightPanelContext.Provider value={contextValue}>
          <ThreatIntelligenceOverview />
        </RightPanelContext.Provider>
      </TestProviders>
```
on top of this file and call it everywhere around here:)

Reply:
good idea, done!